scripts/gen_eval_files.lua

```diff
@@ -243,7 +243,8 @@ local function get_api_meta()
   end
 
   for _, fun in pairs(functions) do
-    local deprecated = fun.deprecated_since ~= nil
+    -- TODO(justinmk): why isn't `fun.deprecated_since` set for nvim_subscribe?
+    local deprecated = fun.deprecated_since ~= nil or fun.deprecated
```

Comment:
Without this change, `@deprecated` is not added to the `nvim_subscribe` doc in `runtime/lua/vim/_meta/api.lua`. Why isn't `fun.deprecated_since` set for `nvim_subscribe`?